modules/460-log-shipper/hooks/internal/vector/destination.go

```diff
@@ -123,15 +125,15 @@ func NewLokiDestination(name string, cspec v1alpha1.ClusterLogDestinationSpec) i
 
 	// default labels
 	labels := map[string]string{
-		"namespace":  "{{ namespace }}",
-		"container":  "{{ container }}",
-		"image":      "{{ image }}",
-		"pod":        "{{ pod }}",
-		"node":       "{{ node }}",
-		"pod_ip":     "{{ pod_ip }}",
-		"stream":     "{{ stream }}",
-		"pod_labels": "{{ pod_labels }}",
-		"pod_owner":  "{{ pod_owner }}",
+		"namespace":    "{{ namespace }}",
+		"container":    "{{ container }}",
+		"image":        "{{ image }}",
+		"pod":          "{{ pod }}",
+		"node":         "{{ node }}",
+		"pod_ip":       "{{ pod_ip }}",
+		"stream":       "{{ stream }}",
+		"pod_labels_*": "{{ pod_labels }}",
```

Comment:
Why do we need asterisk here?

Reply:
This is all about the vector patch
https://github.com/vectordotdev/vector/pull/12041

Reply:
O, yes. I saw your [pull request](https://github.com/vectordotdev/vector/pull/12041) and this is ok